Assets/Scripts/Interactables/Dialogue/DialogueInteractable.cs

```diff
@@ -6,20 +6,20 @@ public class DialogueInteractable : Interactable
 {
     public Conversation conversation;
 
-    private GameObject NPCBubble;
-    private GameObject BirdieBubble;
-    private SpeakerUI speakerUILeft;
-    private SpeakerUI speakerUIRight;
+    private GameObject npcBubble;
+    private GameObject birdieBubble;
+    private SpeakerUI speakerUIBirdie;
+    private SpeakerUI speakerUINPC;
 
     private int activeLineIndex = 0;
 
     public override void OnInteract(GameObject birdie)
     {
-        BirdieBubble = birdie.transform.GetChild(1).gameObject; 
-        NPCBubble = this.transform.GetChild(1).gameObject;
+        birdieBubble = birdie.transform.GetChild(1).gameObject; 
+        npcBubble = this.transform.GetChild(1).gameObject;

```

Comment:
We aren't using these so we can remove it now right?

Reply:
yes !! sorry my bad haha i'm removing both now :)